extensions/amp-lightbox-gallery/0.1/amp-lightbox-gallery.css

```diff
@@ -23,27 +23,22 @@
   z-index: 2147483642;
 }
 
+.amp-lbg-mask {
```

Comment:
let's move "public" css classes to the top of the file with a comment stating they are public and can be changed.